### PR TITLE
Two caches

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Separates into a dynamic and a static cache
- Have to test this with minecolonies yet
-

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/structures/client/BlueprintHandler.java [hunk 1]
```diff
@@ -19,12 +20,15 @@
     /**
      * A static instance on the client.
      */
-    private static final BlueprintHandler ourInstance = new BlueprintHandler();
-    private static final int CACHE_SIZE = 30;
-    private static final long CACHE_EVICT_TIME = 45_000L;
+    private static final BlueprintHandler ourInstance      = new BlueprintHandler();
+    private static final int              CACHE_SIZE       = 30;
```
Comment (Nightenom) [inline src/main/java/com/ldtteam/structures/client/BlueprintHandler.java:24]: Use smaller size for static and something like 15sec (15000ms) evict time. So it doesn't eat so much

File: src/main/java/com/ldtteam/structures/client/BlueprintHandler.java [hunk 2]
```diff
@@ -83,16 +109,29 @@ public void draw(final Blueprint blueprint, final BlockPos pos, final MatrixStac
      */
     public void cleanCache()
     {
-        final long now = System.currentTimeMillis();
-        final Iterator<Int2LongMap.Entry> iter = evictTimeCache.int2LongEntrySet().iterator();
+        long now = System.currentTimeMillis();
```
Comment (Nightenom) [inline src/main/java/com/ldtteam/structures/client/BlueprintHandler.java:112]: we can only use one "now"

**Review by Nightenom — APPROVED** [review]
Remove changes from other PRs

File: src/main/java/com/ldtteam/structures/client/BlueprintHandler.java (hunk 2 above)
Comment (Raycoms) [reply src/main/java/com/ldtteam/structures/client/BlueprintHandler.java:112]: I might be "overdoing it", wondered if so much time passed after all of the above, to create a new one lower.

**Pull request closed without merging**